packages/openactive-integration-tests/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "app.js",
   "scripts": {
     "start": "echo \"Error: package contains only tests\" && exit 1",
-    "test": "jest",
+    "test": "tsc && jest",
```

Comment:
Given that "test" is the command test suite users will use, do we want to have a separate script for this? We can then include it in the GitHub Actions CI script as an explicit step?

Reply:
Ah, good point @nickevansuk. I'll do that

Reply:
This is somewhat counter to the usual node.js pattern for packages (where `npm test` is expected by CI tools to check that the repo can be published e.g. run lint, run unit tests, etc) but I'm happy to keep with the established pattern

Reply:
@nickevansuk I've moved it to a separate `static-checks` script. I'll set it to run in a GitHub action in a separate PR

Reply:
Interesting... do we want to put the tests under `npm start` and put static checks under `npm test`, would that be clearer?

Reply:
This would mean that the checks would be performed by any CI automatically without need of CI-specific config like GitHub actions. I think I agree with this. I can do it in another PR?

Reply:
Yep, perfect - and we'll need to update the docs too as it's a breaking change